include/spm.rst

```diff
@@ -47,12 +47,14 @@ Peripherals configured as Non-Secure
    * REGULATORS
    * RTC0, RTC1
    * SAADC
-   * SPIM3
+   * SPI1-3
    * TIMER0-2
-   * TWIM2
-   * UARTE0, UARTE1
+   * TWI1-3
+   * UARTE1, UARTE3
    * WDT
 
+If you want to turn UARTE0, TWI0, EGU0 and SPI0 non-secure ensure that they are not used by the SPM / Zephyr instance which SPM is using to setup the SPU.
```

Comment:
```suggestion
If you want to configure UARTE0, TWI0, EGU0, or SPI0 as Non-Secure, you must ensure that they are not used by the library instance that the :ref:`secure_partition_manager` sample uses to set up the SPU.
```

Not sure if that suggestion is technically correct though. Are we referring to this library with "SPM" here? Or to the sample?

Reply:
The sample/ the firmware which is running the SPM.

If you change the security attribute to either of the peripherals while being in this sample you'll get a bus fault as the addressing has changed from what you compiled with.

Reply:
Sorry - I meant the SPM library or the SPM sample. 
But I *think* that's what my suggestion says? ;-)

Reply:
I think the order is reversed so you must ensure that they are not used by the sample. 

If we think of the SPM sample as consumer of the SPM library. Then the person configuring it with the SPM sample must ensure that it does not touch any of the peripherals which is being set to non-secure. If not since it's compiled for secure it will use the wrong addressing for peripherals.

Reply:
I'll try to incorporate your suggestion in the reversed order.